frontend/src/components/workspaces/Grid/WorkspaceGridItem/WorkspaceGridItem.tsx

```diff
@@ -18,7 +18,9 @@ const WorkspaceGridItem: FC<IWorkspaceGridItemProps> = ({ ...props }) => {
           className={`cursor-pointer row shadow-lg h-24 w-24 2xl:h-28 2xl:w-28 workspaceitem ${
             isActive ? 'active' : ''
           }`}
-          onClick={() => onClick(id)}
+          onClick={() => {
```

Comment:
If it is active I would do nothing -> call the onClick only if it is not active

Reply:
fixed